files/fcron.conf.in

```diff
@@ -23,3 +23,7 @@ sendmail	=	@@SENDMAIL@
 
 # Location of the default editor for "fcrontab -e"
 editor		=	@@FCRON_EDITOR@
+
+# Display name for the "From: " header of mails sent by us. Watch out! It
+# might cause rejections from picky SMTPDs.
```

Comment:
Can you add something to say fcron follows Vixie cron's default email header format, but it isn't standard rfc2822 so might get rejected from picky SMTPDs ?

Reply:
Actually, the "From" header is now compliant with RFC5322 (which obsoletes 2822). So the caveat about picky SMTPDs should be removed. I'd keep only the Vixie mention, is it fine with you?

Reply:
actually can you confirm whether the 'mailbox' address is identical to what it was before your change if the user doesn't explicitly set and change `displayname` ?

Reply:
Well, not really: 
* old behavior: 'mailbox' addr = `foo@bar (fcron)` -- breaks the RFC,
* new behavior:  'mailbox' addr = `DISPLAYNAME <foo@bar>`.

With empty DISPLAYNAME, it'd give  'mailbox' addr = `foo@bar`.  
The closest we can get to the old behavior is by having a default `displayname = (fcron)`, which gives 'mailbox' addr = `"(fcron)" <foo@bar>` (notice the double quotes).
 
Otherwise, we can just fix the whole affair with a sane hardcoded mailbox addr `"(fcron)" <...>`, and drop the configurable displayname (that's maybe a bit overkill) ^^.

What do you think?